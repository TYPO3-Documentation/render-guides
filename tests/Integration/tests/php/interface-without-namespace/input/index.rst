==============
PHP Interface
==============

..  php:interface:: TestInterface

    Lorem Ipsum Dolor!
