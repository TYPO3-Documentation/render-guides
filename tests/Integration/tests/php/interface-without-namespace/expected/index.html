<!-- content start -->
    <div class="section" id="php-interface">
    <h1>PHP Interface</h1>

            <dl class="php interface">
    <dt class="sig sig-object php"
        id="testinterface">
        <em class="property"><span class="pre">interface</span> </em>
        <span class="sig-name descname"><span class="pre">TestInterface</span></span>

    </dt>
    <dd>
        <p>Lorem Ipsum Dolor!</p>
    </dd>
</dl>

    </div>

<!-- content end -->